diff --git a/packages/opencode/src/session/prompt.ts b/packages/opencode/src/session/prompt.ts
index bf939c7e2..3655f52dd 100644
--- a/packages/opencode/src/session/prompt.ts
+++ b/packages/opencode/src/session/prompt.ts
@@ -561,11 +561,11 @@ export namespace SessionPrompt {
       const agent = await Agent.get(lastUser.agent)
       const maxSteps = agent.steps ?? Infinity
       const isLastStep = step >= maxSteps
-      msgs = await insertReminders({
-        messages: msgs,
-        agent,
-        session,
-      })
+      // msgs = await insertReminders({
+      //   messages: msgs,
+      //   agent,
+      //   session,
+      // })
 
       const processor = SessionProcessor.create({
         assistantMessage: (await Session.updateMessage({
